docqueries/version/doc-full_version.result

```diff
@@ -6,7 +6,7 @@ SET
 SELECT version, library FROM pgr_full_version();
   version  |     library
 -----------+-----------------
- 3.4.0-dev | pgrouting-3.4.0
+ 4.0.0-dev | pgrouting-4.0.0
```

Comment:
I think the develop branch has v3.4.0

Reply:
ah, that is why reviews are needed. one it